source/playgrounds.txt

```diff
@@ -121,6 +122,25 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+   .. tab:: Your Project File
+      :tabid: project-file
+
+      1. Open a project file.
+
+         To open a file in |vscode-short|, click the file name in the
+         file explorer. 
+
+      #. Highlight the code that you want to open in a playground.
+
+      #. Right-click to select :guilabel:`Export Code to Playground`.
+
+         Right-click your selected code to open the options menu.
+         Click :guilabel:`MongoDB Copilot Participant` >
+         :guilabel:`Export Code to Playground`. 
+
+         When you click :guilabel:`Export Code to Playground`, |vscode-short|
+         opens a new playground with your selected code. 
```

Comment:
Let's specify that this opens a new MongoDB playground file with your query exported to MongoDB Shell syntax. You can then hit the run button in the top left corner to test this query with the cluster you are currently connected to in your MongoDB extension. You can iterate on your query in the playgrounds until you are happy with your query. Once the query looks good to you, you can export it back to your application code by using the Export to Language feature (link to Export to Language page).

Reply:
I will add the note about exporting to language with the Export to Language PR (since this one will be merged first)